library-integration-slack/gradle.properties

```diff
@@ -7,3 +7,5 @@ logRecorderVersion = 0.3.3
 mockitoKotlinVersion = 2.0.0-RC1
 springBootVersion = 2.0.4.RELEASE
 feignVersion = 9.7.0
+wiremockVersion = 2.15.0
```

Comment:
If you rely on the inheritance via `spring-cloud-contract-wiremock` this version does not need to be specified explicitly.

Reply:
fixed